costar_dataset/block_stacking_reader_torch.py

```diff
@@ -705,7 +773,7 @@ def ConvertImageListToNumpy(data, format='numpy', data_format='NHWC'):
                     raise ValueError('CostarBlockStackingDataset: Trying to open something which is not a file: ' + str(example_filename))
                 with h5py.File(example_filename, 'r') as data:
                     if 'gripper_action_goal_idx' not in data or 'gripper_action_label' not in data:
-                        raise ValueError('block_stacking_reader.py: You need to run preprocessing before this will work! \n' +
+                        raise ValueError('CostarBlockStackingDataset: You need to run preprocessing before this will work! \n' +
                                          '    python2 ctp_integration/scripts/view_convert_dataset.py --path ~/.keras/datasets/costar_block_stacking_dataset_v0.4 --preprocess_inplace gripper_action --write'
```

Comment:
hmm, do we have view_convert_dataset in the costar dataset repository? Is it independent enough that we could move it here?

Reply:
I don't think view_convert_dataset is really needed in the costar_dataset repo. 
We will most likely run the preprocessing script before we upload the dataset, so this situation will likely never happen.
We could move the script in, but it would add additional dependencies for the package.